src/components/Modal/Modal.module.scss

```
@@ -0,0 +1,47 @@
+@import "../../styles/themes";
+
+.container,
+.overlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
```

can you use [`inset`](https://css-tricks.com/almanac/properties/i/inset/)?
```suggestion
  inset: 0;
```

---

👍